### ISSUE TITLE
Ground-truth answer for OpenThoughts3 questions

### ISSUE THREAD

**Issue by RulinShao** [open-thoughts/open-thoughts#40]
Hi! Thanks for your amazing work! I noticed the released OpenThoughts3 training data does not come with ground truth answer even for verifiable questions, which is quite important for potential follow up that require accurate verification. I wonder if you have any plan to release the ground-truth answers as well? Thanks!

**Comment by EtashGuha** [comment]
Hi Rulin. Unfortunately, these questiosn don't have "ground truth" answers or at least human verified problems. I would say maybe extractng the majority answer from each of 16 responses for each problem would prob be pretty close or better than human verified problems. Thanks!

**Comment by RulinShao** [comment]
Sounds great! I was wondering if any of the subsets in the original datasets came with labels. Do you recommend using MV labels from OT3 or switching to OT2 which came with labels if I want to do RL with it? Thanks for your reply!

**Comment by EtashGuha** [comment]
I'd stick with MV. The ground truth labels in OT2 have similarish problems. If you have the compute, I'd try both! 😅

**Comment by RulinShao** [comment]
Gotcha! Thanks! This is super helpful!
